translations/en.json

```diff
@@ -1,5 +1,5 @@
 {
-  "login": "Login",
```

Comment:
Looks like the babel plugin automatically deleted the existing translations, which is a bug in the babel plugin. See https://github.com/openmrs/openmrs-esm-login/blob/master/.babelrc#L11 and https://i18next-extract.netlify.com/#/configuration?id=discardoldkeys.

I had noticed that the discardOldKeys option wasn't perfect. Maybe let's turn it off? Also, though we should definitely reinstate the `login`, `username`, and `password` keys.

Reply:
They came back :man_shrugging: . I'm happy to continue trying it. Maybe it'll become clear what's weird about it and we can bugfix.